examples/streamz/README.md

```diff
@@ -22,128 +22,144 @@ Run the following to build the image:
---label_file /path/to/label.json \
---poll_interval 1s \
---max_batch_size 1000 \
---data /path/to/dataset
+docker run -it \
+    -p 9787:8787 \
+    -p 9888:8888 \
+    -v <your_volume_binding_host_directory_path>:<your_volume_binding_container_directory_path> \
+    --gpus '"device=0,1,2"' \
+    --name clx_streamz \
+    -d clx-streamz:latest
 ```
 
 **Legacy - Docker CE v18 and nvidia-docker2**
+
 ```
 docker run -it \
---runtime=nvidia \
--p 8787:8787 \
---name cybert-streamz
--d cybert-streamz:latest \
---broker localhost:9092 \
---group_id streamz \
---input_topic input \
---output_topic output \
---model_file /path/to/model.pth \
---label_file /path/to/label.json \
---poll_interval 1s \
---max_batch_size 1000 \
---data /path/to/dataset
+    -p 9787:8787 \
+    -p 9888:8888 \
+     -v <your_volume_binding_host_directory_path>:<your_volume_binding_container_directory_path> \
+    --runtime=nvidia \
+    --name clx_streamz \
+    -d cybert-streamz:latest
 ```
 
-**Parameters:**
-- `broker`* - Host and port where kafka broker is running
-- `group_id`* - Kafka [group id](https://docs.confluent.io/current/installation/configuration/consumer-configs.html#group.id) that uniquely identifies the streamz data consumer.
-- `input_topic` - The name for the input topic to send the input dataset. Any name can be indicated here.
-- `output_topic` - The name for the output topic to send the output data. Any name can be indicated here.
-- `model_file` - The path to your model file
-- `label_file` - The path to your label file
-- `poll_interval`* - Interval (in seconds) to poll the Kafka input topic for data
-- `max_batch_size`* - Max batch size of data (max number of logs) to ingest into streamz with each `poll_interval`
-- `data` - The input dataset to use for this streamz example. This is a filepath to text file containing lines of text to be processed for inference (i.e. log file)
-
-``*`` = More information on these parameters can be found in the streamz [documentation](https://streamz.readthedocs.io/en/latest/api.html#streamz.from_kafka_batched).
-
+The Dockerfile contains an ENTRYPOINT which calls [entrypoint.sh](https://github.com/rapidsai/clx/blob/branch-0.17/examples/streamz/scripts/entrypoint.sh) to:
+1. Configure and install Kafka
+2. Run Kafka broker on `localhost:9092` and Zookeeper on `localhost:2181`
+3. Creates (cyBERT and DGA detection) specific input and output kafka topics and publishes sample input data 
 
-The Dockerfile contains an ENTRYPOINT which calls [entry.sh](https://github.com/rapidsai/clx/blob/branch-0.16/examples/streamz/scripts/entry.sh) to:
-1. Download, install and configure Kafka
-2. Run Kafka and Zookeeper
-3. Create input and output Kafka topics
-4. Read input data into input Kafka topic
-5. Start cyBERT distributed inference using Dask (one worker per GPU)
+Your Quickstart Docker container includes the data and models required to run cyBERT and DGA detection stream processing workflows.
```

Comment:
Is it possible to run both of these at the same time? I'm assuming from this that it would require multiple containers. If that's the case, might want to be a bit more explicit that you can choose your workflow.

Reply:
Sure, will add more details.

Reply:
@BartleyR we can run both the workflows at same time in one container.

Reply:
Updated